pyproject.toml

```diff
@@ -13,3 +13,8 @@ dependencies = [
     "brian2==2.9.0",
     "setuptools",
 ]
+
+[project.scripts]
+brian2wasm = "brian2wasm.__main__:main"
+
+packages = ["brian2wasm"]
```

Comment:
Could you remove this line, please? `packages` should not appear under `project.scripts`